package.json

```diff
@@ -39,16 +39,18 @@
 	"scripts": {
 		"test": "npx htest ./test",
 		"dtslint": "dtslint --expectOnly types",
-		"lint": "eslint . --fix --ext .js --ext .ts --ext .js --ext .cjs",
+		"eslint": "eslint . --ext .js --ext .ts --ext .cjs",
+		"lint": "run-s \"eslint -- --fix\" dtslint",
+		"lint:ci": "run-s \"eslint -- --max-warnings 0\" dtslint",
```

Comment:
Locally `npm run lint` will pass, but display warnings. In CI, lint warnings will cause CI to fail.